CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.19.5-1
+
+* Add `ngx_http_gzip_module` configuration.
+* Add `server_names_hash_bucket_size` configuration.
```

Comment:
I left out `server_names_hash_bucket_size` as it's only necessary to tweak if we have a large number of upstreams, which is not likely to be the case when this is going to be a sidecar.

Reply:
I see. Fair enough.